### ISSUE TITLE
latest Install gives error

### ISSUE THREAD

**Issue by VillaCare** [Roomify/RfA#412]
Made a brand new install with the latest release and now have the following;

An AJAX HTTP request terminated abnormally. Debugging information follows. Path: http://villacare.holiday/install.php?profile=roomify&welcome=done&locale=en&roomify_product=accommodations&dev_style=off&id=1&op=do StatusText: ResponseText: ReadyState: 4

## Expected Behavior
a clear new install

## Current Behavior
error appeared

## Possible Solution
???

## Steps to Reproduce from a clean installation of the latest release
Complete new install of latest release, new DB

**Comment by acrollet** [comment]
Are there any errors in the logs? Our installation tests are functioning normally...

**Comment by VillaCare** [comment]
Dear Adrian,

I think I am better giving up to work with Roomify or to get it to work. 

Over the years I have set it up to a good site than 1 update ruined the whole page. Ever since then I tried to install it again and again, spend money to a prof to get the site as we wanted than a hack or virus destroyed it.

I have no clue why I can not get it to work, don’t know if it is the program or the 1and1 hosting that I can not properly get it installed as one and the other claim on their side all is OK but it does work which is a pitty as I had thought to have found a program I liked so much to be the basis of our business.

Kind regards,

Ferdinand van Reijsen
VillaCare
HTTP://www.villacare.eu
